### PR TITLE
add log sampling at the Functions layer

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: src/api-service/__app__/host.json [hunk 1]
```diff
@@ -5,26 +5,21 @@
...
-    },
     "logLevel": {
-      "default": "Information",
-      "Host.Results": "Information",
+      "default": "Warning",
+      "Host": "Warning",
       "Function": "Information",
-      "Function.FunctionA": "Information",
-      "Host.Aggregator": "Information"
+      "Host.Aggregator": "None",
```
Comment (ranweiler) [inline src/api-service/__app__/host.json:12]: Maybe keep error-level here?

Comment (ranweiler) [reply src/api-service/__app__/host.json:12]: Or warning.

File: src/api-service/__app__/host.json [hunk 2]
```diff
@@ -5,26 +5,21 @@
...
-      "Host.Results": "Information",
-      "Function": "Information",
-      "Host.Aggregator": "Information"
-    },
     "logLevel": {
-      "default": "Information",
-      "Host.Results": "Information",
+      "default": "Warning",
+      "Host": "Warning",
       "Function": "Information",
```
Comment (ranweiler) [inline src/api-service/__app__/host.json:11]: 👍

**Review by ranweiler — APPROVED** [review]

File: src/api-service/__app__/host.json (hunk 1 above)
Comment (bmc-msft) [reply src/api-service/__app__/host.json]: moved to warning

**Pull request merged**